packages/core/utils/index.js

```diff
@@ -0,0 +1,10 @@
+// Node 8 supports native async functions - no need to use compiled code!
+exports.promisify =
+  parseInt(process.versions.node, 10) < 8
+    ? require('./lib/promisify')
+    : require('./src/promisify');
+
+exports.errorUtils =
```

Comment:
heh, we should try to find a way to make this easier if there are going to be a lot of utils 😉